### PR TITLE
MM-17212 Add E2E to test interactive dialog without element

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add E2E to test interactive dialog without element

#### Ticket Link
Jira ticket - [MM-17212](https://mattermost.atlassian.net/browse/MM-17212)

### CODE REVIEW

File: e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        // # Login as sysadmin and ensure that teammate name display setting us set to default 'username'
+        cy.apiLogin('sysadmin');
+        cy.apiSaveTeammateNameDisplayPreference('username');
+
+        // # Create new team and create command on it
+        cy.apiCreateTeam('test-team', 'Test Team').then((teamResponse) => {
+            const team = teamResponse.body;
+            cy.visit(`/${team.name}`);
+
+            const webhookBaseUrl = Cypress.env().webhookBaseUrl;
```
Comment (thekiiingbob) [inline e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: This jumped out as wrong to me, but I actually assume it should work? Cypress docs say to do `Cypress.env('foobar')` to grab variables, but without an argument it returns an object of all env vars which is why your approach works. No change required, just found it interesting!

Comment (saturninoabril) [reply e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: Yes, that works.  Can't remember where I first saw it but I like such approach (but my preference only).  We can change it or document on our style guide if we found it anti-pattern or something that degrades test performance.

File: e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        const newSettings = {
+            ServiceSettings: {
+                AllowedUntrustedInternalConnections: 'localhost',
+                EnablePostUsernameOverride: true,
+                EnablePostIconOverride: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login as sysadmin and ensure that teammate name display setting us set to default 'username'
```
Comment (prapti) [inline e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: Typo: .. display setting **is** set to...?

Comment (saturninoabril) [reply e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: Good catch, will update that

File: e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+        // * Verify that the interactive dialog modal open up
+        cy.get('#interactiveDialogModal').should('be.visible').within(() => {
+            // * Verify that the header of modal contains icon URL, title and close button
+            cy.get('.modal-header').should('be.visible').within(($elForm) => {
+                cy.get('#interactiveDialogIconUrl').should('be.visible').and('have.attr', 'src', simpleDialog.dialog.icon_url);
+                cy.get('#interactiveDialogModalLabel').should('be.visible').and('have.text', simpleDialog.dialog.title);
+                cy.wrap($elForm).find('button.close').should('be.visible').and('contain', '×').and('contain', 'Close');
+
+                cy.get('#interactiveDialogModalLabel').should('be.visible').and('have.text', simpleDialog.dialog.title);
```
Comment (prapti) [inline e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: Is this line repeated intentionally? Looks like a duplicate of line 70.

Comment (saturninoabril) [reply e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js]: Oh no, done removing this line

**Pull request merged**